public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -775,7 +775,7 @@ figure.image-display
   If you're coding along with the steps in this reactive forms tutorial, 
   create the pertinent files based on the 
   [source code displayed below](#source-code "Reactive Forms source code"). 
-  Notice that `hero-list.component.ts` and `hero-list.component.ts` 
+  Notice that `hero-list.component.ts` and `hero.service.ts` 
   import `Observable` and `finally` from `rxjs`. 
```

Comment:
Yes, I think you've proposed the most accurate version. I've corrected my PR in accordance with your changes.